app/assets/stylesheets/components/_grid.sass

```diff
@@ -8,3 +8,11 @@ $Grid-margin-base: $Theme-spacing-base
   display: inline-block
   margin-left: $Grid-margin-base
   margin-top: $Grid-margin-base
+  vertical-align: top
+
+.Grid-2-columns
```

Comment:
Besides there are other ways of getting this behavior. We'll discuss them tomorrow